docs/GCP-development.md

```diff
@@ -20,181 +20,202 @@ for Python and Kind and/or GCP/GKE.
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:98 Loading version '0.0.1' for pluggable type 'hashedrekord'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:97 Loading support for pluggable type 'rpm'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:98 Loading version '0.0.1' for pluggable type 'rpm'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:97 Loading support for pluggable type 'jar'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:98 Loading version '0.0.1' for pluggable type 'jar'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:97 Loading support for pluggable type 'alpine'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:98 Loading version '0.0.1' for pluggable type 'alpine'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:97 Loading support for pluggable type 'tuf'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:98 Loading version '0.0.1' for pluggable type 'tuf'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:97 Loading support for pluggable type 'rekord'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:98 Loading version '0.0.1' for pluggable type 'rekord'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:97 Loading support for pluggable type 'intoto'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:98 Loading version '0.0.1' for pluggable type 'intoto'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:97 Loading support for pluggable type 'rfc3161'
+2022-02-23T18:29:56.397Z        INFO    app/serve.go:98 Loading version '0.0.1' for pluggable type 'rfc3161'
+2022-02-23T18:29:56.401Z        INFO    storage/storage.go:40   Configuring attestation storage at file:///var/run/attestations
+2022-02-23T18:29:56.420Z        INFO    restapi/server.go:234   Serving rekor server at http://[::]:3000
+```
+
+We didn't deploy the sigstore pieces with ingress so let's use kubectl port-forward to make requests to rekor and fulcio
 
-### Adding localhost entries to make tools usable
+```bash
+kubectl port-forward svc/fulcio 8081:80 -n fulcio-system &
+kubectl port-forward svc/rekor-server 8080:3000 -n rekor-system &
+```
 
 Add the following entries to your `/etc/hosts` file
-
 ```
 127.0.0.1 rekor.rekor-system.svc
 127.0.0.1 fulcio.fulcio-system.svc
 127.0.0.1 ctlog.ctlog-system.svc
 ```
 
-This makes using tooling easier, for example:
+Testing Fulcio
 
-```shell
-rekor-cli --rekor_server http://rekor.rekor-system.svc:8080 loginfo
+```bash
+curl -s http://fulcio.fulcio-system.svc:8081/api/v1/rootCert |  openssl x509 -noout -subject
+subject= /O=Example, Org./CN=Example Authority
 ```
 
-For example, this is what I get after smoke tests have successfully completed:
+This will match the Subject from Terraform Google CA deployment 
 
-```shell
-rekor-cli --rekor_server http://rekor.rekor-system.svc:8080 loginfo
+Testing Rekor
+
+```bash
+ curl http://rekor.rekor-system.svc/api/v1/log/
```

Comment:
does this need to be:
rekor.rekor-system.svc:8080
?

Reply:
The port forwards `kubectl port-forward svc/rekor-server 8080:3000 -n rekor-system &` and the /etc/host make that possible.